Apps/GatewayApi/test/unit/GatewayApiTests.csproj

```diff
@@ -5,6 +5,7 @@
     <SonarQubeTestProject>true</SonarQubeTestProject>
     <RootNamespace>HealthGateway.GatewayApiTests</RootNamespace>
     <DocumentationFile>bin\$(Configuration)\$(TargetFramework)\$(MSBuildThisFileName).xml</DocumentationFile>
+    <RunSettingsFilePath>$(MSBuildProjectDirectory)\.runsettings</RunSettingsFilePath>
```

Comment:
It doesn't look like the .runsettings file was included in the commit.

Reply:
Removed from the project.